pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go

```diff
@@ -116,12 +116,12 @@ func (c *EtcdMembersController) reportEtcdMembers() error {
 		}
 	}
 
-	if len(notStartedMembers) != 0 {
+	if len(notStartedMembers) != 0 || len(availableMembers) != 3 {
```

Comment:
this will break adding 4th master, we don't support it, but still doesn't feel good breaking it in control loop

Reply:
I believe that statically setting this to `3` will also further complicate issues that have cropped up building a single node cluster for code ready containers.  

See:  https://bugzilla.redhat.com/show_bug.cgi?id=1805034 

If three is the magic number for production ready clusters, (which I wholeheartedly believe that it is), then perhaps we need a switch that can be set when invoking the installer to override off this restriction for single node clusters.  Or, even a way to override it completely for those who are brave, or just really want 4+ nodes in their control plane...

Reply:
FWIW, we're also trying to get CRC working for OKD 4.X with FCOS as well.

https://github.com/openshift/okd/issues/88